src/Domain/CoordinateMaps/TimeDependent/ShapeMapTransitionFunctions/SphereTransition.cpp

```diff
@@ -9,23 +9,23 @@
 #include "Domain/CoordinateMaps/TimeDependent/ShapeMapTransitionFunctions/SphereTransition.hpp"
 #include "Utilities/ConstantExpressions.hpp"
 #include "Utilities/ContainerHelpers.hpp"
+#include "Utilities/ErrorHandling/Error.hpp"
 
 namespace domain::CoordinateMaps {
 
 SphereTransition::SphereTransition(double r_min, double r_max)
-    : r_min_(r_min),
-      r_max_(r_max),
-      a_(-1. / (r_max - r_min)),
-      b_(r_max / (r_max - r_min)) {
+    : r_min_(r_min), r_max_(r_max) {
   if (r_min <= 0.) {
     ERROR("The minimum radius must be greater than 0 but is " << r_min);
   }
-  if (r_max <= r_min) {
+  if (r_max < r_min) {
```

Comment:
I think removing the `=` was my comment, but it's wrong, isn't it? The `if (r_max <= r_min) then error` was correct, no? I beg for your forgiveness and kindly ask you revert this